ui/src/app/pages/workspace/create-billing-account-modal.spec.tsx

```diff
@@ -15,124 +21,86 @@ describe('CreateBillingAccountModal', () => {
-    ).toBeTruthy();
-    expect(
-      wrapper
-        .find('[data-test-id="user-phone-number"]')
-        .first()
-        .prop('disabled')
-    ).toBeFalsy();
+    expect(getByTestId('user-first-name')).toBeDisabled();
+    expect(getByTestId('user-last-name')).toBeDisabled();
+    expect(getByTestId('user-contact-email')).toBeDisabled();
+    expect(getByTestId('user-workbench-id')).toBeDisabled();
+    expect(getByTestId('user-institution')).toBeDisabled();
+    expect(getByTestId('user-phone-number')).not.toBeDisabled();
 
-    expect(
-      wrapper.find('[data-test-id="user-first-name"]').first().prop('value')
-    ).toEqual('Tester!@#$%^&*()><script>alert("hello");</script>');
-    expect(
-      wrapper.find('[data-test-id="user-last-name"]').first().prop('value')
-    ).toEqual('MacTesterson!@#$%^&*()><script>alert("hello");</script>');
-    expect(
-      wrapper.find('[data-test-id="user-phone-number"]').first().prop('value')
-    ).toBeUndefined();
-    expect(
-      wrapper.find('[data-test-id="user-contact-email"]').first().prop('value')
-    ).toEqual('tester@mactesterson.edu><script>alert("hello");</script>');
-    expect(
-      wrapper.find('[data-test-id="user-workbench-id"]').first().prop('value')
-    ).toEqual('tester@fake-research-aou.org');
-    expect(
-      wrapper.find('[data-test-id="user-institution"]').first().prop('value')
-    ).toEqual(BROAD.displayName);
+    expect(getHTMLInputElementValue(getByTestId('user-first-name'))).toEqual(
+      'Tester!@#$%^&*()><script>alert("hello");</script>'
+    );
+    expect((getByTestId('user-last-name') as HTMLInputElement).value).toEqual(
+      'MacTesterson!@#$%^&*()><script>alert("hello");</script>'
+    );
+    expect(getHTMLInputElementValue(getByTestId('user-phone-number'))).toEqual(
+      ''
+    );
+    expect(getHTMLInputElementValue(getByTestId('user-contact-email'))).toEqual(
+      'tester@mactesterson.edu><script>alert("hello");</script>'
+    );
+    expect(getHTMLInputElementValue(getByTestId('user-workbench-id'))).toEqual(
+      'tester@fake-research-aou.org'
+    );
+    expect(getHTMLInputElementValue(getByTestId('user-institution'))).toEqual(
+      BROAD.displayName
+    );
 
     // Type invalid phone number then valid phone number
-    wrapper
-      .find('[data-test-id="user-phone-number"]')
-      .first()
-      .simulate('change', { target: { value: '1234x' } });
-    expect(
-      wrapper.find('[data-test-id="next-button"]').first().prop('disabled')
-    ).toBeTruthy();
-    wrapper
-      .find('[data-test-id="user-phone-number"]')
-      .first()
-      .simulate('change', { target: { value: '1234567890' } });
-    expect(
-      wrapper.find('[data-test-id="next-button"]').first().prop('disabled')
-    ).toBeFalsy();
+    fireEvent.change(getByTestId('user-phone-number'), {
+      target: { value: '1234x' },
+    });
+    expectButtonElementDisabled(screen.getByRole('button', { name: /next/i }));
```

Comment:
Updated to use our button helper function that Copilot is unaware of:
https://docs.google.com/document/d/1LDxH33OzrWM0-ZpgNrR5WQb8yG5AmaKqd7cZlLTo1v0/edit#heading=h.ilt9yzsj16k9